tutorials/working_with_auxiliary_data.md

```diff
@@ -0,0 +1,89 @@
+# Working with auxiliary data
+
+This tutorial gives an overview of how to deal with auxiliary information in recommender systems using Cornac. In our context, auxiliary data stands for information beyond user-item interactions or preferences, which often hold a clue on how users consume items. Examples of such information or modalities are: item textual descriptions, user/item reviews, product images, social network, etc.
+
+## Modality classes and utilities
+
+In addition to implementing readers and utilities for different types of data, the `cornac.data` module provides modality classes, namely GraphModality, ImageModality and TextModality. The purpose of the latter classes is to make it convinient to work with the corresponding modalities by:
```

Comment:
In addition to implementing readers and utilities for different types of data, `cornac.data` module provides modality classes, namely `GraphModality`, `ImageModality` and `TextModality`. The purpose of the latter classes is to make it `convenient` to work with the corresponding modalities by: